doc/fonts.md

```diff
@@ -0,0 +1,51 @@
+# Fonts management
+
+## Fonts source
+Fonts are assets which are downloaded from separate repositories using `download_assets.py` script (see more: [download assets documentation](../doc/download_assets.md)). Basing on configuration (Community/Proprietary) fonts will be downloaded from two different repositories.
+
+## Fonts mapping
+Regardless of the source, both assets packages contain file fontmap.json, mapping font files to font types used in MuditaOS code.
+
+Example:
+```
+{
+    "info" : {
+        "default_font_name" :       "dejavu_sans_bold_27",
+        "default_font_type_name" :  "dejavu_sans"
+    },
+    "style": {
+        "mediumboldtitle":      "dejavu_sans/dejavu_sans_bold_27.mpf",
+        "huge":                 "dejavu_sans/dejavu_sans_regular_140.mpf",
+        "supersizeme":          "dejavu_sans/dejavu_sans_regular_90.mpf",
+        "supersizemelight":     "dejavu_sans/dejavu_sans_light_90.mpf",
+        "largelight":           "dejavu_sans/dejavu_sans_light_46.mpf",
+        "large":                "dejavu_sans/dejavu_sans_regular_46.mpf",
+        "verybiglight":         "dejavu_sans/dejavu_sans_light_38.mpf",
+        "verybig":              "dejavu_sans/dejavu_sans_regular_38.mpf",
+        "mediumbigbold":        "dejavu_sans/dejavu_sans_bold_32.mpf",
+        "bigbold":              "dejavu_sans/dejavu_sans_bold_30.mpf",
+        "big":                  "dejavu_sans/dejavu_sans_regular_30.mpf",
+        "biglight":             "dejavu_sans/dejavu_sans_light_30.mpf",
+        "mediumbold":           "dejavu_sans/dejavu_sans_bold_27.mpf",
+        "medium":               "dejavu_sans/dejavu_sans_regular_27.mpf",
+        "mediumlight":          "dejavu_sans/dejavu_sans_light_27.mpf",
+        "smallbold":            "dejavu_sans/dejavu_sans_bold_24.mpf",
+        "small":                "dejavu_sans/dejavu_sans_regular_24.mpf",
+        "verysmallbold":        "dejavu_sans/dejavu_sans_bold_20.mpf",
+        "verysmall":            "dejavu_sans/dejavu_sans_regular_20.mpf"
+    }
```

Comment:
consider just linking to existing example in code :bow: and just pasting here one or two exemplary lines